resources/language/resource.language.pl_pl/strings.po

```diff
@@ -89,92 +90,299 @@ msgctxt "#30044"
 msgid "Debug"
 msgstr "Debug"
 
+msgctxt "#30045"
+msgid "Off"
+msgstr "Wyłącz"
+
+msgctxt "#30046"
+msgid "To 15s before end"
+msgstr "Do 15s. przed końcem"
+
+msgctxt "#30047"
+msgid "To popup start time"
+msgstr "Do czasu rozpoczęcia wyskakującego okienka"
+
+msgctxt "#30048"
+msgid "To detector start time"
+msgstr "Do czasu uruchomienia detektora"
+
 msgctxt "#30049"
-msgid "Next Episode"
+msgid "Next video"
 msgstr "Następny odcinek"
 
+msgctxt "#30050"
+msgid "Disabled"
+msgstr "Nieaktywne"
+
+msgctxt "#30051"
+msgid "Forced on"
+msgstr "WYmuszenie włączone"
+
+msgctxt "#30052"
+msgid "Forced off"
+msgstr "Wymuszenie wyłączone"
 
 ### SETTINGS
 msgctxt "#30500"
-msgid "Interface"
+msgid "Appearance"
 msgstr "Wygląd"
 
 msgctxt "#30501"
-msgid "Features"
-msgstr "Funkcje"
+msgid "Popup display"
+msgstr "Wygląd okienka"
+
+msgctxt "#30502"
+msgid "Skin customisation [COLOR gray](requires skin integration)[/COLOR]"
+msgstr "Dostosowanie skóry [COLOR gray](wymaga integracji ze skórką)[/COLOR]"
 
 msgctxt "#30503"
-msgid "Set the display mode for the notifications"
-msgstr "Ustaw tryb wyświetlania powiadomień"
+msgid "Show simple style popup"
+msgstr "Pokaż proste okienko"
+
+msgctxt "#30504"
+msgid "Apply Kodi skin to UpNext popup"
+msgstr "Zastosuj skórkę Kodi do okienka UpNext"
 
 msgctxt "#30505"
 msgid "Show a \"Stop\" button instead of a \"Close\" button"
-msgstr "Pokaż przycisk \"Stop\" zamiast przycisku \"Zamknij\""
+msgstr "Pokaż przycisk \"Zatrzymaj"\" zamiast przycisku \"Zamknij\""
```

Comment:
Additional double quote needs to be removed from this line:
`msgstr "Pokaż przycisk \"Zatrzymaj"\" zamiast przycisku \"Zamknij\""`

Should just be:
`msgstr "Pokaż przycisk \"Zatrzymaj"\ zamiast przycisku \"Zamknij\""`